File: app/utils/dawproject_python/expressionType.py
from enum import Enum


class ExpressionType(Enum):
    GAIN = "gain"
    PAN = "pan"
    TRANSPOSE = "transpose"
    TIMBRE = "timbre"
    FORMANT = "formant"
    PRESSURE = "pressure"
    CHANNEL_CONTROLLER = "channelController"
    CHANNEL_PRESSURE = "channelPressure"
    POLY_PRESSURE = "polyPressure"
    PITCH_BEND = "pitchBend"
    PROGRAM_CHANGE = "programChange"

File: app/utils/dawproject-decorators/expressionType.ts
/** Represents the type of an expression. */
export enum ExpressionType {
  gain = "gain",
  pan = "pan",
  transpose = "transpose",
  timbre = "timbre",
  formant = "formant",
  pressure = "pressure",

  // MIDI
  channelController = "channelController",
  channelPressure = "channelPressure",
  polyPressure = "polyPressure",
  pitchBend = "pitchBend",
  programChange = "programChange",
}
